com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/service/DependencyServiceTest.java

```diff
@@ -45,53 +47,77 @@
 /** This class represent all the unit test for the copybook dependency managament service */
 @Slf4j
 public class DependencyServiceTest extends FileSystemConfiguration {
+  public static final String NESTED_CPY_NAME = "CPYNEST2";
   private final DataBusBroker broker =
       Guice.createInjector(new DatabusModule()).getInstance(DataBusBroker.class);
```

Comment:
I would prefer to avoid using Guice in tests and call the observer callback directly, but it is not critical at the moment and we may do it later

Reply:
I agree to postpone, since many unit test classes are affected by this problem. We can wrap all together in a TD userstory.